### ISSUE TITLE
[PATCH] Typo in deferreds HOWTO page

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7208]
|<img alt="norfolk's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| norfolk reported|
|-|-|
|Trac ID|trac#483|
|Type|defect|
|Created|2004-01-19 17:04:52Z|

Attachments:

* [defer.xhtml.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/9ee/9ee0df7c8a647ff45e95abfac028ef5651be0ac5/b5d7fe87562bbecc5794b8611154f58e96c12338.diff) (470 bytes) - added by norfolk on 2004-01-19 17:04:52Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__483 483
type__defect defect
reporter__norfolk norfolk
priority__high high
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__ 
keywords__documentation documentation
time__1074531892000000 1074531892000000
changetime__1470877064017252 1470877064017252
version__ 
owner__ 
cc__radix cc__hypatia cc__norfolk
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="norfolk's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|norfolk commented|
|-|-|

```
#!html
<pre>
In the fourth code sample on the deferreds HOWTO page, the comments are wrong
(the sample produces a ValueError for even numbers, but the comments say that
the odd number should produce an error).
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix commented|
|-|-|

```
#!html
<pre>
someone may have fixed this issue in another way already... at least, running
the example from deferred.xhtml now matches up with what the HOWTO says.

i'm resolving it.
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_4"></a>|GitHub commented|
|-|-|

In changeset a5d6577764ca0170934b06567c3cd6991b6f8ea5
```
#!CommitTicketReference repository="" revision="a5d6577764ca0170934b06567c3cd6991b6f8ea5"
Merge pull request #483 from twisted/8741-rodrigc-dist-py3

Author: rodrigc
Reviewer: alex
Fixes: #8741

Add test_socket and test_time to dist3.py
```
